lightly/models/examples/moco_system.py

```diff
@@ -0,0 +1,67 @@
+import pytorch_lightning as pl
+import torch
+
+from lightly.models.modules.heads import MoCoProjectionHead
+
+import lightly
+from lightly.models.modules.momentum_encoder import MomentumEncoder
+
+
+class MocoSystem(pl.LightningModule):
```

Comment:
I would keep calling it MocoSystem for the moment as the name MoCo is already taken by https://github.com/lightly-ai/lightly/blob/8dde4d68c0598758bfd05c05f225bf754d9f1ed5/docs/source/getting_started/benchmarks/cifar10_benchmark.py#L144

Reply:
I am also in favor of **not** calling it `MoCoSystem`.